go.mod

```diff
@@ -205,7 +197,7 @@ replace (
 	github.com/hashicorp/terraform-exec v0.16.1 => github.com/tf-controller/terraform-exec v0.15.1-0.20220809152546-4850a69faedb
 
 	// Fix CVE-2023-32731
-	google.golang.org/grpc => google.golang.org/grpc v1.53.0
+	google.golang.org/grpc => google.golang.org/grpc v1.56.3
```

Comment:
I would move this out of the replace directive, as I don't understand why we need to pin this to an earlier version. Alternatively if we *have* to pin it to a specific version, I would updated the comment above to indicate which CVE this addresses.